.storybook/main.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     stories: ['../**/*.stories.tsx'],
     webpackFinal: async (config, { configType }) => {
         config.module.rules.push({
-            test: /\.tsx?$/,
+            test: /\.[jt]sx?$/,
```

Comment:
Can remove?

Reply:
no, looks like it is needed to compile dependencies. Copyied the regex from DCR, it supports JS/TS/JSX/TSX